core/tests/protractor_desktop/creatorDashboard.js

```diff
@@ -123,4 +124,107 @@ describe('Creator dashboard functionality', function() {
     expect(creatorDashboardPage.getSubscribers()).toEqual('2');
     users.logout();
   });
+
+  it('should work fine in grid view as well as list view', function() {
+    var feedback = 'A good exploration. Would love to see a few more questions';
+    // Create required users.
+    users.createUser(
+      'user1@creatorDashboard.com',
+      'creatorDashboardOwner');
+    users.createUser(
+      'user2@creatorDashboard.com',
+      'learner2');
+    users.createUser(
+      'user3@creatorDashboard.com',
```

Comment:
These users are the same as the above test right? If so we should move it to a beforeAll block I think.

Reply:
I think the users should be different here. It would give error since the user is already registered with the same account. (Didn't notice since I ran the test not the suite)

I'm not using beforeAll because the test will have dependencies -- feedbacks, plays, views of previous test would also account into this test. Making this as independent looks cleaner to me. What do you say?

Reply:
Oh good point. I agree

Reply:
Updated the users login credentials!